### DISCUSSION TITLE
External Monitors on Laptop

### DISCUSSION THREAD

**Discussion by NunoGomes-Coding** [ChefKissInc/NootedRed#162] [Q&A] [answered]
Hi,
Starting by saying thanks to the developer, you're doing an amazing job.
I got an HP Omen, with an amd cpu and an nvidia dgpu. I get no display on any ports I have, including the usb-c dp alt mode that is connected to the igpu. I was wondering if is it possible to somehow patch or drive the outputs so we can get external displays working on laptops, similar to WhateverGreen does with intel iGPUs. Doing my research I couldn't find any info on this.

Any more info required please let me now, ill leave here below my IOReg and EFI.
[EFI.zip](https://github.com/ChefKissInc/NootedRed/files/12538321/EFI.zip)
[ioreg.zip](https://github.com/ChefKissInc/NootedRed/files/12538446/ioreg.zip)

**Comment by VisualEhrmanntraut** [comment, marked as answer]
If you have a dGPU, then the port is wired to it and you can't get it working.
